.travis.yml

```diff
@@ -1,10 +1,11 @@
-
----
 language: python
 python:
 - 2.7
 services: docker
 before_install:
+- openssl aes-256-cbc -K $encrypted_4096e639581d_key -iv $encrypted_4096e639581d_iv
+  -in pipelines/data-generation/travis-secrets/secrets.zip.enc -out pipelines/data-generation/travis-secrets/secrets.zip -d
+- unzip pipelines/data-generation/travis-secrets/secrets.zip -d pipelines/data-generation/travis-secrets/
```

Comment:
might be better to put the file into `geni/` and name it `cloudlab_credentials.zip`, this with the goal of being explicit about what's in the files.